cli/go/src/pxf-cli/cmd/cluster.go

```diff
@@ -39,13 +39,13 @@ func createCobraCommand(use string, short string, cmd *command) *cobra.Command {
 
 var (
 	clusterCmd  = createCobraCommand("cluster", "Perform <command> on each segment host in the cluster", nil)
-	initCmd     = createCobraCommand("init", "Initialize the PXF server instance and install PXF extension under $GPHOME on master, standby master, and all segment hosts", &InitCommand)
+	initCmd     = createCobraCommand("init", "(deprecated) Install PXF extension under $GPHOME on master, standby master, and all segment hosts", &InitCommand)
 	startCmd    = createCobraCommand("start", "Start the PXF server instances on all segment hosts", &StartCommand)
 	stopCmd     = createCobraCommand("stop", "Stop the PXF server instances on all segment hosts", &StopCommand)
 	statusCmd   = createCobraCommand("status", "Get status of PXF servers on all segment hosts", &StatusCommand)
 	syncCmd     = createCobraCommand("sync", "Sync PXF configs from master to standby master and all segment hosts. Use --delete to delete extraneous remote files", &SyncCommand)
-	resetCmd    = createCobraCommand("reset", "Reset PXF (undo initialization) on all segment hosts", &ResetCommand)
-	registerCmd = createCobraCommand("register", "Install PXF extension under $GPHOME on all segment hosts", &RegisterCommand)
+	resetCmd    = createCobraCommand("reset", "(deprecated) No operation", &ResetCommand)
```

Comment:
since this is only deprecated and not removed, it seems like it is still an operation you can perform (and there are still locations elsewhere. (See  cli/go/src/pxf-cli/cmd/pxf.go line 184) that describe what this does. Should we leave the description of this command much like we do for init? 

```suggestion
	resetCmd    = createCobraCommand("reset", "(deprecated) Reset PXF (undo initialization) on all segment hosts", &ResetCommand)
```

Reply:
Well, pxf reset will not do anything actually, we keep the command, but plan to remove it at a later date